src/Components/Details/Details.css

```diff
@@ -128,25 +136,18 @@
   .details_main-text {
     width: 261px;
   }
-
-  
 }
 
 @media only screen and (min-width: 700px) {
-  .modal-container {
-    display: flex;
-    align-items: center;
+  .details_main {
+    width: 95vw;
```

Comment:
vw refers to the viewport so I would recommend the main be 95% of its parent, we can change the grid-area arrangement of the app if we want the details_main to cover more area

Reply:
This change is now made - thanks for the recommendation!